src/main/java/org/jabref/gui/importer/actions/OpenDatabaseAction.java

```diff
@@ -102,11 +103,20 @@ public static void performPostOpenActions(ParserResult result, DialogService dia
 
     @Override
     public void execute() {
-        FileDialogConfiguration fileDialogConfiguration = new FileDialogConfiguration.Builder()
+        FileDialogConfiguration.Builder builder = new FileDialogConfiguration.Builder()
                 .addExtensionFilter(StandardFileType.BIBTEX_DB)
-                .withDefaultExtension(StandardFileType.BIBTEX_DB)
-                .withInitialDirectory(getInitialDirectory())
-                .build();
+                .withDefaultExtension(StandardFileType.BIBTEX_DB);
+
+        FileDialogConfiguration fileDialogConfiguration;
+        try {
+            fileDialogConfiguration = builder
+                    .withInitialDirectory(getInitialDirectory())
+                    .build();
+        } catch (IllegalArgumentException e) { // something went wrong with the directory, fallback to user directory
```

Comment:
Yep thats my bad, misread the stacktrace and didnt check it again.

What test cases are you looking for to test withInitialDirectory? There are already test cases present and it seems like they cover all cases? Or are you looking for a test case specific to the directory returned by `Directories#getUserDirectory()`

Reply:
Mabye add a test case for a mounted path? how does it look like and why does it fail?

Reply:
The catch needs to be around the open/show methhod

Reply:
I'm not sure its possible to simulate a mounted directory in an automated test.

Correct me if I'm wrong, but since mounted directories are handled at the os-level and are treated identically to any other path at the application level, a path with a successful mount is no different to any other directory inside of a unit test, meaning that even if it was possible to simulate one in code, it wouldn't be any different to a regular path?

Did you instead mean that I should test for a *failed* mounted path?
I'm pretty sure I can use mockito for this to simulate an error being thrown by `org.jabref.gui.DialogService#showFileOpenDialogAndGetMultipleFiles(org.jabref.gui.util.FileDialogConfiguration)` when it gets called to replicate what happened in the original issue, but I don't see a way that I can write a useful test for `org.jabref.gui.util.FileDialogConfiguration.Builder#withInitialDirectory(java.nio.file.Path)` as requested.

Reply:
@Siedlerchr

Reply:
@Boston54 Did you try with https://github.com/google/jimfs?

Reply:
> Yep thats my bad, misread the stacktrace and didnt check it again.

Did you move the block? Mayb,e we can let this through without any additonal test and blindly moving on. But the code should make the impression it could fix the issue.

Reply:
@koppor I was looking through jimfs and I've added it to the project, but to me it seems as though it isn't useful for simulating a true mounted directory (but you can 'simulate it' by basically just having a normal path that uses mount naming conventions and using your imagination).
I'm still using it though in the test I've written along with mockio to simulate what happened in the original issue (`showFileOpenDialogAndGetMultipleFiles()` throwing an `IllegalArgumentException`).

And yes, I've moved the block to be around the correct area. Since you are okay with not having a test that specifically simulates the mounted directory, I'll have a final check over it soon and then push it.